accepted/00000-ptf-repos-in-uyuni.md

```diff
@@ -0,0 +1,104 @@
+- Feature Name: PTF repositories in Uyuni
+- Start Date: 2021-05-17
+
+# Summary
+[summary]: #summary
+
+SUSE want to provide customers PTFs (Product Temporary Fixes) in repositories.
```

Comment:
Do I get correctly that this will only be for PTFs and not TEST packages, right?

Reply:
I asked back last week. The answer is "no". Also TEST patches should be delivered in these repos.
For SUSE there is a mechanism which should prevent to install it automatically.
But that requires a change in the solver which we get only for libzypp.
This means we need to talk about other OSes in this regards.

I will start a new section about handling PTFs. This is part 2 of Project Lotus, but may need to be included in the reposync discussion.

Reply:
@mcalmer what do we need to talk to other OSs? to what OSs? AFAIK the ptf() is only used by SUSE operating systems :-?

Reply:
For Debian and Ubuntu, we can use `Dpkg::Pre-Invoke` to check whether the package is a PTF and ignore it in that case

https://wiki.debian.org/AptConfiguration#apt.conf